bittensor/_cli/cli_impl.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, config: 'bittensor.Config' ):
                 config (:obj:`bittensor.Config`, `required`): 
                     bittensor.cli.config()
         """
-        if not config.no_version_checking:
+        if "no_version_checking" in config and config.no_version_checking:
```

Comment:
`if not config.get("no_version_checking", True):`

would be better.

Reply:
Kinda, although you need to read the docs on the get function to understand the boolean